tekton_configs/test-run.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  name: pipelinerun-testing
+  namespace: gabrielwen
+spec:
+  pipelineSpec:
+   tasks:
+   - name: xgboost-synthetic
+     taskRef:
+      name: pytest
+     params:
+     - name: src-dir
+       value: /mnt/testdir/src/gabrielwen
+     - name: repos
+       value: kubeflow/examples@HEAD,kubeflow/testing@HEAD,kubeflow/tf-operator@HEAD
```

Comment:
Do we need tf-operator?

Reply:
I'm not really sure - we can remove this and try when we start implementing things.